### DISCUSSION TITLE
any kind of metadata in png

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#2750] [Q&A] [answered]
any kind of metadata in png? or at leat a txt file per png ?

**Comment by ILikeToasters** [comment, marked as answer]
You can enable Metadata in the debug settings and you can adjust your config file to always embed the metadata. Otherwise you can look at the history log. Each output folder for each day will have it's own log file in the output folder.  Also this gives more info:  https://youtu.be/SopP2GNJwz8?si=wPhELqKDCJGNdJs9&t=540
<img width="337" alt="Screenshot 2024-04-11 105334" src="https://github.com/lllyasviel/Fooocus/assets/4357657/c26a6df9-390f-4bfb-9df9-0e1e75bba1e2">

**Comment by kalle07** [comment]
thy for the hint  ...
you know the line for config.txt ? fooocus doesnt save any kind of options and it give no save button  ,,,,  A HINT for a featue  ;)

**Reply by ILikeToasters** [reply]
Examples that can be used in the config.txt file are found in the config_modification_tutorial.txt file.  For example you would use 
```
"default_save_metadata_to_images": true,
"default_metadata_scheme": "fooocus",
```

**Comment by kalle07** [comment]
thx...

maybe another question
is it possible to name the files i save, default is only image(1) 2 3 and so on
useful is maybe date incl sec plus seed.

**Reply by ILikeToasters** [reply]
No.

**Reply by kalle07** [reply]
maybe a feature ?

**Reply by ILikeToasters** [reply]
It could be added if someone wanted to do it but I am not sure if there is really any demand. I know for me the timestamps for filenames work perfectly.

**Reply by kalle07** [reply]
you say timestamps, fooocus saves timestamps? if i save/download (iam not shure which) its only named: filname1 2 3

**Reply by ILikeToasters** [reply]
Yes. It sounds like you are saving the images through the UI. But all images that are generated are saved automatically in the output folder which by default you will find in the Fooocus folder. Each day will have a separate folder and also a log file in each folder for all the images.

**Reply by ILikeToasters** [reply]
More info: https://youtu.be/HWCmuxqz7pw?si=Pve2jc-4tYD86wPX&t=78

**Reply by kalle07** [reply]
ah i see, but i dont want repeat watch all images ... i want save only which are good but not with filname1  ^^
